src/components/DatePicker.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import DateFnsUtils from '@date-io/date-fns';
+import 'date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+import PropTypes from 'prop-types';
+import exact from 'prop-types-exact';
+
+function DateField(props) {
```

Comment:
*DatePicker

Reply:
oops don't know how that slipped. I fixed it in the latest push (: